src/main/resources/static/css/home-page-style.css

```diff
@@ -0,0 +1,81 @@
+body{
```

Comment:
Prefer space between selector and brace
```suggestion
body {
```